crates/payload/validator/src/lib.rs

```diff
@@ -159,6 +159,12 @@ impl ExecutionPayloadValidator {
             }
         }
 
+        if self._is_prague_active_at_timestamp(sealed_block.timestamp) &&
```

Comment:
do you mind renaming this method as well to not have the underscore?

Reply:
Good catch!